data/human/human missions.txt

```diff
@@ -5900,7 +5900,7 @@ mission "Quicksilver Mixup 2"
 	on offer
 		conversation
 			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
-			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
+			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month; some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```

Comment:
Em-dash would probably be better, since `some mechanical issues with his ship, I'm told` doesn't stand up alone.

```suggestion
			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month - some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```

Reply:
Isn't that just a regular dash?

Reply:
Yeah, it is. Copy/pasted an em dash and put it in.

Reply:
Ah, sorry, let me explain:

ES currently only supports ASCII (because Unicode support is a total mess).
Grammatically, an em-dash is what belongs here, so the "right" thing to do would be to put one in (`—`), probably (but not necessarily) without a space on either side. Unfortunately, we can't because it won't render, so the standard in ES is to use a hyphen padded with spaces on each side (` - `) instead, which incidentally looks almost identical on GitHub.

Reply:
Oh, okay. I'll switch it out with a hyphen, then.